### DISCUSSION TITLE
How to change super admin password

### DISCUSSION THREAD

**Discussion by michaeljcoyne** [area17/twill#1408] [General]
Ho do I change the super admin password created by  php artisan twill:superadmin?

The user does not show up in the user listing.

Thannks

Michael

**Comment by pboivin** [comment]
Hi @michaeljcoyne, this is normal, superadmin users are not displayed in the CMS users listing.

For a password reset, I think you can use the link from the login form, even for superadmin accounts. A more manual approach would be through `php artisan tinker`, something like :

```
$user = User::find(1);
$user->password = Hash::make('mypassword');
$user->save();
```

Hope this can help!
